docs/dev-tools/zombienet/tutorials/minimal-example.md

```diff
@@ -0,0 +1,234 @@
+
+```typescript
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+async function main() {
+  const wsProvider = new WsProvider('ws://127.0.0.1:55308');
+    const api = await ApiPromise.create({ provider: wsProvider });
+
+    // Retrieve the chain & node information via rpc calls
+    const [chain, nodeName, nodeVersion] = await Promise.all([
+        api.rpc.system.chain(),
+        api.rpc.system.name(),
+        api.rpc.system.version()
+    ]);
+
+    console.log(`You are connected to chain ${chain} using ${nodeName} v${nodeVersion}`);
+}
+
+main().catch(console.error).finally(() => process.exit());
+```
+
+Either way allows you to interact easily with the network and its nodes.
+
+### Checking Metrics
+
+You can also check the metrics of the nodes by accessing the provided links listed by the output as `Prometheus Link`. [Prometheus](https://prometheus.io/){target=_blank} is a monitoring and alerting toolkit that collects metrics from the nodes. By accessing the provided links, you can see the metrics of the nodes in a web interface.
+
+![Prometheus metrics for Bob’s node from the Zombienet test](/polkadot-ecosystem-docs-draft/images/dev-tools/zombienet/tutorials/zombienet-1.webp)
+
+### Checking logs
+
+To check the nodes’ logs, you can use the provided command listed by the output as 'Log Cmd'. For instance, to check the logs of the `alice` node, you can open a new terminal and run the following command:
+
+```bash
+tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-SEzfCidQ1za4/alice.log
+```
+
+After running this command, you will see the logs of the `alice` node in real-time, which can be useful for debugging purposes. The logs of the `bob` and `collator01` nodes can be checked similarly.
+
+## Running a Test
+
+To run a test against the spawned network, you can use the [Zombienet DSL](../testing.md) to define the test scenario. For example, you can create a file named `minimal-example-test.zndsl` with the following content:
+
+```toml
+Network: ./minimal-config.toml
+Creds: config
+
+alice: is up
+alice: parachain 100 is registered within 225 seconds
+alice: parachain 100 block height is at least 10 within 250 seconds
+
+bob: is up
+bob: parachain 100 is registered within 225 seconds
+bob: parachain 100 block height is at least 10 within 250 seconds
+
+# metrics
+alice: reports node_roles is 4
+alice: reports sub_libp2p_is_major_syncing is 0
+
+bob: reports node_roles is 4
+bob: reports substrate_network_gossip_expired_messages_total is 835
+
+collator01: reports node_roles is 4
+collator01: reports substrate_sub_libp2p_listeners_local_addresses is 2
+```
+
+This test scenario checks to verify the following:
```

Comment:
This is not rendering as a bullet list when I spin it up locally. Please double check the needed line returns are in there, etc. For me it's showing up as a single paragraph with the dashes in where the new bullet line should start.

Reply:
Sorry @dawnkelly09 , what do you mean by "check if the necessary line breaks are there"? I'm not sure I follow

Anyway, I'm making a small change to simplify this minimal test even further

Reply:
Sometimes I forget to put a blank line between the sentence and the first item in the bullet list. It makes it all come out as one big sentence with the dashes in it like so: 

"- the nodes are running - the parachain with ID 100 is registered within a certain timeframe (255 seconds in this example) - the parachain block height is at least a certain number within a timeframe (in this case, 10 within 255 seconds)- the nodes report metrics"

So just make sure there is a blank line between the the sentence and the `-the nodes are running`